README.md

```diff
@@ -215,6 +215,55 @@ This function returns a list of dataframes. Each dataframe is the standings for
 5      Cincinnati Reds   68  94  .420  35.5
 ```
 
+# Caching
+
+To facilitate faster data retrieval for repeated calls, a local data cache may be used to save a local copy of the
+requested data. By default the cache is disabled so as to respect a user's potential desire to not have their hard drive
+space used without their permission. However, enabling the cache is simple.
+
+Cache can be turned on by including the pybaseball.cache module and enabling the cache option like so:
+
+```python
+from pybaseball import cache
+
+cache.enable()
+```
+
+This will store a copy of the data returned for each DataFrame returning function call.
+This data will be stored in `~/.pybaseball/cache` (or `%USERPROFILE%\.pybaseball\cache` in Windows) by
+default. This can be configured by modifying the config:
+
+```python
+from pybaseball import cache
+
+cache.cache_config.cache_directory = '~/other_location'
+```
+
+The cache_config has many options that can be set or it can be replaced with a new CacheConfig object:
+
+```python
+from datetime import timedelta
+
+from pybaseball import cache
+
+cache.cache_config = cache.CacheConfig(
+   enabled=True,
+   cache_directory='~/other_other_location',
+   expiration=timedelta(days=7),
+   cache_type=cache.CacheType.CSV
+)
+```
+
+Each CacheConfig option is as follows:
+- `enabled` - bool - Whether cache should be enabled
+- `cache_directory` - str - The location to store the cached data. If it does not exist, it will be created.
+- `expiration` - datetime.timedelta - The timedelta after the cache file is created to expire it. Default = 24 hours.
+- `cache_type` - pybaseball.cache.CacheType - The method to use in storing the cache. Options:
+  - `CacheType.CSV` - Cache is stored in pandas compatible CSV format files
+  - `CacheType.PARQUET` - Cache is stored in Apache Parquet format files: https://parquet.apache.org/
+  - `CacheType.PICKLE` - Cache is stored as python pickle files
```

Comment:
We really don't need an enum for 3 values

Reply:
or 2

Reply:
Makes sense. I'll tweak.

Reply:
And now down to CSV and PARQUET, and those are just strings (validated with a Literal type by mypy).